sig-scalability/throughput_burts_slo.md

```diff
@@ -0,0 +1,23 @@
+# SLO: Kubernetes cluster of size at least X is able to start Y Pods in Z minutes
+
+## Burst Pod Startup Throughput SLO
+### Summary
+Kubernetes cluster of size at least X is able to start Y Pods in Z minutes.
+### User Stories
+- User is running a workload of X total pods and wants to ensure that it can be started in Y time. 
+- User wants to understand how quickly they can react to a dramatic change in workload profile - e.g. they need to begin scheduling new pods when QPS meets W% of max to accommodate a spike due to a Black Friday Sale.
```

Comment:
What will the tests choose for this value? How is QPS metric being fed to scheduler?